lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -219,6 +220,39 @@
                 },
 
                 /**
+                 * Render the edit team form.
+                 */
+                editTeam: function (topicID, teamID) {
+                    var self = this;
+                    this.getTeamsView(topicID).done(function (teamsView) {
+                        self.getTeam(teamID, false).done(function(team) {
+                            self.mainView = new ViewWithHeader({
+                                header: new HeaderView({
+                                    model: new TeamsHeaderModel({
+                                        description: gettext("If you edit any team details, you should notify team members of your changes."),
+                                        title: gettext("Edit Team"),
+                                        breadcrumbs: [
+                                            {
+                                                title: teamsView.main.teamParams.topicName,
```

Comment:
This breadcrumb seems wrong-- it takes you back to the listing of all teams in the topic. I would expect to go back to the Team view.